src/Sylius/Component/Registry/DependencyInjection/Compiler/AliasedServicePass.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+
+namespace Sylius\Component\Registry\DependencyInjection\Compiler;
```

Comment:
Well -- I don't think putting it in a bundle would make much sense really - the bundle would not *do* anything (that I am aware) - not DI config, not templates, not assets. The most appropriate place for this would be, using SF as an example, the `Bridge` namespace. But, the overhead of creating and requiring that ...